main.js

```diff
@@ -23,6 +24,11 @@ let win
 // a global reference of the logger object.
 let logger
 
+// global reference for electron store
```

Comment:
Why do you use global?
What about using remote require there?

Reply:
to make the store available to any other renderer process that might use it. does it provide any performance enhancement to use remote over global?